examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -13,10 +13,9 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion adjacent-strings-literals
   }
 
-  (name, year, birth) {
+  (String name, num year, num birth) {
     // #docregion string-interpolation
-    'Hello, ' + name + '! You are ' + (year - birth) + ' y...';
-
+    'Hello, ' + name + '! You are ' + (year - birth).toString() + ' y...';
```

Comment:
This code excerpt was actually invalid, given that it is assumed `year` and `birth` are numbers (`.toString()` was missing).